net-passwd.h-example

```diff
@@ -9,6 +9,8 @@
 
 // IP Address of the MQTT BROCKER
 #define myMQTT_BROCKER "192.168.1.1"
+#define myMQTT_USERNAME "user"
+#define myMQTT_PASSWORD "pass"
```

Comment:
I would also leave a comment that about how to deal in case of no user and password requirements. Otherwise users might play around with NULL, "" or something like that.